src/util/index.js

```diff
@@ -30,4 +30,18 @@ function emitFirst (emitter, events, handler) {
   })
 }
 
+/**
+ * Converts BufferList messages to Buffers
```

Comment:
Should we add this to `libp2p-utils`? We are doing similar conversions in other repos